app/views/spree/admin/payments/source_views/_paypal.html.haml

```diff
@@ -0,0 +1,36 @@
+%fieldset{"data-hook" => "paypal"}
+  %legend{align: "center"}= Spree.t(:transaction, scope: :paypal)
+  .row
+    .alpha.six.columns
+      %dl
+        %dt
+          = Spree.t(:payer_id, scope: :paypal)
```

Comment:
Ideally all these keys would be out of the spree namespace.

Reply:
I think I'd prefer to do that separately, there's already a lot of changes going on in this PR. 

It would also force translators to update all of the related strings, right?